# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../portmap/errno-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN portmap_4-orig/Makefile portmap_4/Makefile
--- portmap_4-orig/Makefile	1996-05-31 15:50:40.000000000 +0200
+++ portmap_4/Makefile	2003-03-05 16:04:05.000000000 +0100
@@ -50,7 +50,8 @@
 
 # Auxiliary object files that may be missing from your C library.
 #
-AUX	= daemon.o strerror.o
+AUX	= daemon.o 
+#strerror.o
 
 # NEXTSTEP is a little different. The following seems to work with NS 3.2
 #
diff -urN portmap_4-orig/portmap.c portmap_4/portmap.c
--- portmap_4-orig/portmap.c	1996-05-31 15:52:59.000000000 +0200
+++ portmap_4/portmap.c	2003-03-05 16:14:49.000000000 +0100
@@ -85,6 +85,7 @@
 #include <stdio.h>
 #include <syslog.h>
 #include <netdb.h>
+#include <errno.h>
 #include <sys/socket.h>
 #include <sys/ioctl.h>
 #include <sys/wait.h>
@@ -124,7 +125,7 @@
 static void callit();
 struct pmaplist *pmaplist;
 int debugging = 0;
-extern int errno;
+/*extern int errno;*/
 
 #include "pmap_check.h"
 
